### PR TITLE
fix: #187 - adjusted sidebar height

### PR DESCRIPTION
**Pull request by prasanth7890**
### Before fix - 
image
### After fix - 
image

I adjusted the height of the sidebar from screen height to (screen height - navbar height), which solves the issue of not showing all the list items in sidebar in scrolling.
(Ignore the names on sidebar titles, they were all duplicated just to replicate the issue on local machine)

### CODE REVIEW

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -115,7 +115,9 @@ export function Sidebar({
   }
 
   return (
-    <div className="overflow-y-scroll h-screen w-[300px] min-w-[133px] bg-gray-50 dark:bg-gray-800 cursor-pointer sticky top-[64px] self-start w-84">
+    <div
+     className={`overflow-y-scroll h-[${(window.innerHeight - 64)}px] w-[300px] min-w-[133px] bg-gray-50 dark:bg-gray-800 cursor-pointer sticky top-[64px] self-start w-84`}
```
Comment (devsargam) [inline src/components/Sidebar.tsx:119]: pls, use tailwind utility

Comment (devsargam) [inline src/components/Sidebar.tsx:119]:
```suggestion
     className="overflow-y-scroll max-w-[calc(100vw-64px)] w-[300px] min-w-[133px] bg-gray-50 dark:bg-gray-800 cursor-pointer sticky top-[64px] self-start w-84"
```

**Review by devsargam — CHANGES_REQUESTED** [review]

File: src/components/Sidebar.tsx (hunk 1 above)
Comment (prasanth7890) [reply src/components/Sidebar.tsx]: updated with suggested changes. please review

Comment (devsargam) [reply src/components/Sidebar.tsx]: LGTM

**Review by devsargam — APPROVED** [review]

**Comment by hkirat** [comment]
please add screenshots/description of the change

**Comment by prasanth7890** [comment]
> please add screenshots/description of the change

Hii Harkirat sir, 
i edited the first comment by adding screenshots and changes. once please review. 

Thank you

**Pull request merged**